segmentation/utils/spatial_analysis_utils.py

```diff
@@ -229,7 +230,7 @@ def calculate_enrichment_stats(close_num, close_num_rand):
             z[j, k] = (close_num[j, k] - muhat[j, k]) / sigmahat[j, k]
             # Calculate both positive and negative enrichment p values
             p_pos[j, k] = (1 + (np.sum(tmp >= close_num[j, k]))) / (bootstrap_num + 1)
-            p_neg[j, k] = (1 + (np.sum(tmp <= close_num[j, k]))) / (bootstrap_num + 1)
+            p_neg[j, k] = (1 + (np.sum(tmp < close_num[j, k]))) / (bootstrap_num + 1)
```

Comment:
With regards to the change in the testing of p_neg, changing this from <= to < does the trick for neutral matrices (aka those with the same distribution parameters for distr_AB and distr_random).

Reply:
@Jaiveers21, in the original matlab code, was this comparison >= only for positive, for both, or for neither?

I think the mathematically correct comparison here would be strict >

Reply:
In the matlab, both had the or equal to